types/tx/ext.go

```diff
@@ -0,0 +1,21 @@
+package tx
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec/types"
+)
+
+// TxExtensionOptionI defines the interface for tx extension options
+type TxExtensionOptionI interface{}
```

Comment:
Can you fix the failing test? I think to register `testdata.Cat` into this interface in that particular test.